app/assets/javascripts/controllers/SearchCtrl.coffee

```diff
@@ -0,0 +1,29 @@
+@bruseApp.controller 'SearchCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+  $scope.files = []
+
+  # watches the search input field for changes
+  $scope.$watch "search", () ->
```

Comment:
This function is called when changes are made to the `search`-model so I don't think it's necesary. But we could test if it makes any difference.